lang/python/python-cryptography/Makefile

```diff
@@ -32,7 +32,7 @@ define Package/python-cryptography/Default
   CATEGORY:=Languages
   SUBMENU:=Python
   URL:=https://github.com/pyca/cryptography
-  DEPENDS:=+libopenssl
+  DEPENDS:=+libopenssl +@OPENSSL_WITH_DEPRECATED +@OPENSSL_ENGINE
```

Comment:
where is `OPENSSL_ENGINE` defined ? I can't seem to find it
I can find `OPENSSL_WITH_DEPRECATED` though in `https://github.com/openwrt/openwrt/blob/master/package/libs/openssl/Config.in#L18`.

Reply:
Ah it seems to be related to https://github.com/openwrt/openwrt/pull/965

That pull request seems close to being merged. I expect that it will be done before the next release.

I guess we must wait.